CLIENT-SPECIFICATION.md

```diff
@@ -179,6 +179,8 @@ The [`LANG` environment variable](https://www.gnu.org/software/gettext/manual/ht
 
 The [`LANGUAGE` environment variable](https://www.gnu.org/software/gettext/manual/html_node/The-LANGUAGE-variable.html) specifies a priority list of languages that a user wishes to read in. In absence of `LANG` and if this environment variable is present, a client MUST use the defined priority list to decide on the language of pages to display. If a page is not available in the user's preferred language, then a client MUST respect the user's priority list defined in the `LANGUAGE` variable, and MAY choose to notify the user that a page in their chosen language couldn't be found (perhaps along with a link to the [translations section of the contributing guide](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#translations)).
 
+Regardless of the language selected through the above environment variables, clients MUST always attempt to fallback to English if the page does not exist in the requested languages. In this case clients REALLY SHOULD tell the user that the page does not exist in their requested language however if it was not English. If the client supports a command-line argument for language, the client MUST only attempt to show the page in that language (clients OUGHT TO notify the user that a page is available in other languages if present).
+
```

Comment:
Taking from https://github.com/tldr-pages/tldr/issues/4092#issuecomment-647036499. Could be useful to include a table (similar to the platform & language section) to give examples of different variable configurations to make things as explicit as possible.

```suggestion
LANGUAGE | LANG | Result
---------|------|-------
it,cz,de | cz   | it,cz,de,en
it,de,fr | cz   | it,de,fr,cz,en
  --     | it   | it,en
it,cz    | --   | it,cz,en
  --     | --   | en

Note: `LANG` or `LANGUAGE` may contain a value of `C` or `POSIX` and this should be treated as ignored.
```

Reply:
Should'nt the 4th case in this table have just '"en"' as the Result, since `LANGUAGE` should be ignored, if `LANG` is not set, or set to `"C"`, as described [here](https://www.gnu.org/software/gettext/manual/html_node/The-LANGUAGE-variable.html#The-LANGUAGE-variable)?

Reply:
Yup, updated. All the more reason to have a table as reading those docs is a pain.

Reply:
Thanks for updating and adding this table. It's my first time collaborating here and i'm still learning how things are done.